packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -702,8 +708,10 @@ export const prepareVaultManagerKit = (
                 debtAmount,
                 vault.getCurrentDebt(),
               );
-              const vaultDebt = floorMultiplyBy(debtAmount, debtPortion);
-              const collatPostDebt = AmountMath.subtract(vCollat, vaultDebt);
+              const vaultDebt = ceilDivideBy(debtAmount, price);
```

Comment:
elsewhere in this function "debt" is in the stable token, so I found this a little confusing.
```suggestion
              const collatOwed = ceilDivideBy(debtAmount, price);
```
but not worth another CI build if this one passes

Reply:
done.  changed to `debtInCollateral`.